pages/ksphere/kommander/1.1.0-beta/release-notes/index.md

```diff
@@ -11,56 +11,208 @@ enterprise: false
 
 ## Release Notes
 
-### Version v1.1.0-beta.1 - April 22 March 2020
+<!--
+Template:
+
+### Version <VERSION> - <DATE>
 
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 | **Minimum**        | 1.16.0  |
 | **Maximum**        | 1.16.x  |
 | **Default**        | 1.16.8  |
 
-#### Addon Changelog
+#### Features/Improvements
 
-- added podAnnotations to the kommander-ui chart
-- updated ingress values for kubeaddons-catalog
+- <FEATURES/IMPROVEMENTS>
 
-#### Chart Changelog
+#### Bug Fixes
 
-_Used chart version: `0.5.7` -> `0.6.4`_
+- <FIXES>
+- Lots of smaller UX Bugs and Improvements
 
-- create kubecost ns hook
-- dynamically get grafana service url, add readme file
-- hook should not call static service name
+#### Component Versions
 
-##### Chart Subcomponent Versions
+- Addon: ``
+- Chart: ``
+- KCL: ``
+- UI: ``
+- kommander-karma: ``
+- kubeaddons-catalog: ``
+- kommander-thanos: ``
+- grafana: ``
 
-- KCL: [0.5.1 -> 0.5.2](https://github.com/mesosphere/kommander-cluster-lifecycle/compare/v0.5.1...v0.5.2)
-- UI: [3.16.5 -> 3.25.5](https://github.com/mesosphere/kommander/compare/v3.16.5...v3.25.5)
-- kommander-karma: 0.3.9
-- kubeaddons-catalog: 0.1.6
-- kommander-thanos: 0.1.13
-- grafana: 4.5.1
+-->
 
-### Version v1.1.0-beta.0 - April 8 March 2020
+### Version v1.1.0-rc.1 - May 29th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.16.0  |
```

Comment:
I think we support the minimum version of 1.15.0, according to Jimmi and Dimitri we support current max n-2
```suggestion
| **Minimum**        | 1.15.0  |
```

Reply:
I guess this is a bit unclear… we might need to state 1.15, but it’s technically not supported by kudo, which is part of kommander. 🤷‍♂️
@jaxesn maybe? Can we drop 1.15 here or should we keep it?

Reply:
lets keep it as is for now…